app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,12 @@
+<% provide(:title, "About") %>
```

Comment:
1. remove ()
2. Sử dụng I18n 
ví dụ: `About`
self check toàn bộ project

3. Thực hiện self check và fix tương tự cho các file html.erb nhé